substratevm/src/com.oracle.svm.agent/src/com/oracle/svm/agent/BreakpointInterceptor.java

```diff
@@ -1218,6 +1338,9 @@ public static void onUnload() {
                     brk("java/lang/reflect/Proxy", "newProxyInstance",
                                     "(Ljava/lang/ClassLoader;[Ljava/lang/Class;Ljava/lang/reflect/InvocationHandler;)Ljava/lang/Object;", BreakpointInterceptor::newProxyInstance),
 
+                    brk("sun/reflect/MethodAccessorGenerator", "generateSerializationConstructor",
+                                    "(Ljava/lang/Class;[Ljava/lang/Class;[Ljava/lang/Class;ILjava/lang/Class;)Lsun/reflect/SerializationConstructorAccessorImpl;",
+                                    BreakpointInterceptor::generateSerializationConstructor),
```

Comment:
This PR is only for JDK 8, not tested on JDK 11. Should it support both versions in one patch? I planned to work on JDK 11 after this patch.

Reply:
That would be preferable, we would like to keep the feature set consistent. It might be as simple as just having two optional breakpoints with different signatures (optional so it doesn't matter if one of them cannot be installed).

Reply:
I would actually be more OK with JDK 11 only support, because JDK 11 is more important than JDK 8.
So yes, we do not want to merge a feature like that without JDK 11 support.

Reply:
This seems to work fine:
```
                    optionalBrk("sun/reflect/MethodAccessorGenerator", "generateSerializationConstructor",
                                    "(Ljava/lang/Class;[Ljava/lang/Class;[Ljava/lang/Class;ILjava/lang/Class;)Lsun/reflect/SerializationConstructorAccessorImpl;",
                                    BreakpointInterceptor::generateSerializationConstructor),
                    optionalBrk("jdk/internal/reflect/MethodAccessorGenerator", "generateSerializationConstructor",
                                    "(Ljava/lang/Class;[Ljava/lang/Class;[Ljava/lang/Class;ILjava/lang/Class;)Ljdk/internal/reflect/SerializationConstructorAccessorImpl;",
                                    BreakpointInterceptor::generateSerializationConstructor),
```

Reply:
fixed